draft-ietf-tls-esni.md

```diff
@@ -324,10 +324,10 @@ cipher_suite
 provided in the corresponding `ECHConfig.cipher_suites` list.
 
 config_digest
-: A cryptographic hash of the `ECHConfig` structure from which the ECH key was
-obtained, i.e., from the first byte of "version" to the end of the structure.
-This hash is computed using the hash function associated with `cipher_suite`,
-i.e., the corresponding HPKE KDF algorithm hash.
+: Equal to `Extract(Expand("", config), "ech_config_digest", Nh)`, where
```

Comment:
The order is wrong here -- it should be Expand(Extract(...))

Reply:
ahh, good catch

Reply:
Fixed, here and below.